finagle-mysql/src/main/scala/com/twitter/finagle/mysql/Row.scala

```diff
@@ -20,6 +20,14 @@ trait Row {
   /** The values for this Row. */
   val values: IndexedSeq[Value]
 
+  /**/
```

Comment:
Looks like an unfinished comment.

Fun side note, this broke github's syntax highlighting